components/itemDetailsScreen.brs

```diff
@@ -0,0 +1,42 @@
+sub init()
+    m.itemTitle = m.top.findNode("itemTitle")
+    m.watchButton = m.top.findNode("watchButton")
+    m.itemDescription = m.top.findNode("itemDescription")
+    m.itemTags = m.top.findNode("itemTags")
+    m.itemTime = m.top.findNode("itemTime")
+    m.ButtonGroup = m.top.findNode("ButtonGroup")
+    m.top.ObserveField("content", "handleItemDetails")
+end sub
+
+sub handleItemDetails()
+    m.ButtonGroup.focusedTextFont = "font:LargeBoldSystemFont"
+    m.ButtonGroup.textFont = "font:LargeBoldSystemFont"
+    m.ButtonGroup.vertAlignment = "center"
+    m.ButtonGroup.horizAlignment = "center"
+    m.itemTitle.text = m.top.content.TITLE
+    m.itemDescription.text = m.top.content.DESCRIPTION
```

Comment:
Why `TITLE` and `DESCRIPTION` are capital?

Reply:
They're capital in the documentation, I thought they are supposed to be capital here too.

Reply:
Alright